plugin.js

```diff
@@ -40,13 +41,16 @@ processor.preprocess = (text, filename) => {
 
   const isConfig  = text.startsWith('{') || text.startsWith('[');
   const isHandler = text.startsWith('(') || text.startsWith('function');
+  let prefix = USE_STRICT;
 
   if (isConfig || isHandler) {
     text = MODULE_EXPORTS + text + ';';
+    prefix += MODULE_EXPORTS;
```

Comment:
How about removing the previous line, prepending `';'` to `trail` and changing `USE_STRICT` to `prefix` on line 51?

Reply:
With https://github.com/metarhia/eslint-plugin-impress/pull/7#discussion_r213753539 in mind, not prepending to `trail`, actually, but appending to a new variable `suffix` instead.